src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/SensorClientFragment.kt

```diff
@@ -77,12 +93,24 @@ class SensorClientFragment : Fragment() {
   override fun onStart() {
     super.onStart()
     deviceIdEd.setText(DeviceIdUtil.getLastDeviceId(requireContext()).toString())
+    updateAddress(deviceIdEd.text.toString())
   }
 
   override fun onStop() {
-    super.onStop()
     scope.cancel()
     resetSensorGraph() // reset the graph on fragment exit
+    super.onStop()
+  }
+
+  private fun updateAddress(deviceId: String) {
+    try {
+      ChipClient.getDeviceController(requireContext()).updateDevice(
+          /* fabric ID */ 5544332211,
+          deviceId.toULong().toLong()
```

Comment:
"addressUpdateFragment" is for compressed fabric ID. so, I think you can use "addressUpdateFragment" for fabric ID instead of hard cording.

Reply:
I think this API is outdated anyway as the fabric ID is ignored under the hood (and taken from the controller, so I just added it to silence the compiler. Fabric ID should be removed as a followup from this code and from the addressUpdateFragment.